Editor/SettingsAnalyzers/HDRenderPipelineAnalyzer.cs

```diff
@@ -84,5 +83,16 @@ public ProjectIssue Analyze()
 
             return null;
         }
+
+        void GetCameraComponents(GameObject go, ref List<HDAdditionalCameraData> components)
+        {
+            var comp = go.GetComponent(typeof(HDAdditionalCameraData));
+            if (comp != null)
+                components.Add((HDAdditionalCameraData)comp);
+            for (int i = 0; i < go.transform.childCount; i++)
+            {
+                GetCameraComponents(go.transform.GetChild(i).gameObject, ref components);
+            }
+        }
```

Comment:
Why not using` GetComponentsInChildren()` instead?

Reply:
no particular reason, that would work too.